ros/src/computing/perception/localization/packages/lidar_localizer/launch/ndt_slam.launch

```diff
@@ -0,0 +1,106 @@
+<!-- -->
+<launch>
+
+  <arg name="input_mapping_points_topic" default="/points_raw" />
```

Comment:
can you add documentation to the launch file?
http://wiki.ros.org/roslaunch/XML/arg#Attributes

Reply:
added